packages-content-model/roosterjs-content-model-plugins/lib/edit/EditPlugin.ts

```diff
@@ -31,6 +36,11 @@ export class EditPlugin implements EditorPlugin {
      */
     initialize(editor: IStandaloneEditor) {
         this.editor = editor;
+        this.disposer = this.editor.attachDomEvent({
```

Comment:
I think only Android needs this, so we can add a check for android (you can call editor.getEnvironment().isAndroid)